demos/projects/ST/b-l475e-iot01a/VSCodeDebug.md

```diff
@@ -0,0 +1,51 @@
+# Debug a STMicroelectronics B-L475E-IOT01A Discovery kit using VS Code
+
+## What you need
+
+* Have successfully build the sample for ST Microelectronics DevKit: [B-L475E-IOT01A](https://www.st.com/en/evaluation-tools/b-l475e-iot01a.html)
+
+* Make sure you run this command on Git: As **administrator** run `git config --system core.longpaths true`
```

Comment:
Should this go before the build step since the build will generate long paths?
```suggestion
* Make sure to run this git command before building and cloning:

 ```powershell
 # As **administrator** 
 git config --system core.longpaths true
 ```
```

Reply:
If this indeed is only for Windows this is fine. Otherwise I don't believe Linux needs this so maybe call out for windows